nni/algorithms/compression/pytorch/pruning/iterative_pruner.py

```diff
@@ -554,6 +559,8 @@ class ActivationMeanRankFilterPruner(IterativePruner):
         The activation type.
     sparsity_training_epochs: int
         The number of batches to statistic the activation.
+    statistics_batch_num: int
```

Comment:
why there is no such parameter before?

Reply:
workaround by using `sparsifying_training_batches`